patches/minecraft/net/minecraft/item/MusicDiscItem.java.patch

```diff
@@ -0,0 +1,60 @@
+--- a/net/minecraft/item/MusicDiscItem.java
++++ b/net/minecraft/item/MusicDiscItem.java
+@@ -23,13 +23,12 @@
+ public class MusicDiscItem extends Item {
+    private static final Map<SoundEvent, MusicDiscItem> field_150928_b = Maps.newHashMap();
+    private final int field_195977_c;
+-   private final SoundEvent field_185076_b;
```

Comment:
We don't remove vanilla fields. That's a major binary break, and wouldn't be wanted even if we weren't in the no-breaking-changes stage.

I believe that alone makes this PR in the current state a no-go.

Reply:
This is now outdated, since the field isn't definalized anymore